scripts/runner-env.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+// Update environment variables for all runners in the given environment
+
+const apiUrl = 'https://api.render.com/v1';
+
+if (!process.env.ENVIRONMENT) {
+    console.error('Please set the ENVIRONMENT environment variable');
+    process.exit(1);
+}
+
+if (!process.env.OWNER_ID) {
+    console.error('Please set the OWNER_ID environment variable');
+    process.exit(1);
+}
+
+if (!process.env.RENDER_API_KEY) {
+    console.error('Please set the RENDER_API_KEY environment variable');
+    process.exit(1);
+}
+
+if (!process.env.KEY) {
+    console.error('Please set the KEY environment variable');
+    process.exit(1);
+}
+
+if (!process.env.VALUE) {
+    console.error('Please set the VALUE environment variable');
+    process.exit(1);
+}
+
+console.log('Listing runners...');
+const runners = await fetchRunners();
+console.log(`Found ${runners.length} runners`);
+console.log('Updating env vars...');
+
+for (let i = 0; i < runners.length; i++) {
+    const runner = runners[i];
+    await updateEnvVar(runner.id, process.env.KEY, process.env.VALUE);
+    console.log(`${i + 1} of ${runners.length}: ${runner.name}`);
+}
+
+console.log();
+console.log('Remember: these changes will not take effect until the runner is restarted.');
+console.log();
+
+function sleep(ms) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function fetchRunners() {
+    let services = [];
+    let cursor = '';
+    let running = true;
+
+    while (running) {
```

Comment:
why not `while(true)` and `break` without the `running` var?

Reply:
That was what I had initially and eslint was mad at me. I'll update and put a rule exception there because I think it's easier to read that way anyway.